cmd/trade.go

```diff
@@ -45,10 +46,10 @@ func hiddenFlag(flag string) {
 	}
 }
 
-func logPanic() {
+func logPanic(l logger.Logger) {
 	if r := recover(); r != nil {
 		st := debug.Stack()
-		log.Printf("PANIC!! recovered to log it in the file\npanic: %v\n\n%s\n", r, string(st))
+		l.Errorf("PANIC!! recovered to log it in the file\npanic: %v\n\n%s\n", r, string(st))
```

Comment:
@Reidmcc  can you rebase off the latest master (it already contains some of these changes)

Reply:
yep